InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -78,20 +99,35 @@ public void autonomousInit() {
   public void autonomousPeriodic() {
   }
 
+ @Override
+  public void teleopInit() {
+    comp.start();
+
+  }
+
+
   @Override
   public void teleopPeriodic() {
     teleopController.callStateMachine();  // Also runs drivetrain
     Shooter.getInstance().update();
     Indexer.getInstance().update();
     Intake.getInstance().update();
-    ControlPanel.getInstance().update();
-    ColourSensor.getInstance().update();
+    ControlPanel.getInstance().update();  //FIXME
```

Comment:
remove as this should be called by subsystem

Reply:
Yup, no problem from me!